pep-0008.txt

```diff
@@ -81,10 +81,9 @@ using Python's implicit line joining inside parentheses, brackets and
 braces, or using a *hanging indent* [#fn-hi]_.  When using a hanging
 indent the following should be considered; there should be no
 arguments on the first line and further indentation should be used to
-clearly distinguish itself as a continuation line.
-
-Yes::
+clearly distinguish itself as a continuation line::
 
+    # Correct:
```

Comment:
For this one (only), can you insert a blank line after Correct, and ditto after Wrong? This is because otherwise the scope of the Correct/Wrong is unclear, since there are several blocks with comments.

There are 1-2 others like this that I'll point out explicitly.

Reply:
Ok, I'll do it.